packages/webpack-config-anansi/package.json

```diff
@@ -66,13 +66,13 @@
     "ramda": "^0.26.1",
     "raw-loader": "3.1.0",
     "react-dev-utils": "^9.1.0",
-    "react-docgen-typescript-loader": "^3.3.0",
+    "react-docgen-typescript-loader": "^3.4.0",
     "sass-loader": "^8.0.0",
     "sass-resources-loader": "^2.0.1",
     "stats-webpack-plugin": "^0.7.0",
     "style-loader": "^1.0.0",
     "svg-react-loader": "^0.4.6",
-    "svgo": "^1.3.0",
+    "svgo": "^1.3.2",
```

Comment:
Changelog for reference https://github.com/svg/svgo/blob/master/CHANGELOG.md

![image](https://user-images.githubusercontent.com/6308123/69103409-2f775a80-0a1a-11ea-8ab0-64b74f7ecd82.png)